airflow/cli/commands/connection_command.py

```diff
@@ -59,11 +62,20 @@ def _connection_mapper(conn: Connection) -> Dict[str, Any]:
 @suppress_logs_and_warning
 def connections_get(args):
     """Get a connection."""
+    airflow_console = AirflowConsole()
+    if CUSTOM_BACKEND:
+        airflow_console.print(
+            (
+                "WARNING: The Airflow CLI will not return Connections or "
+                "Variables stored in an alternative secrets backend."
```

Comment:
It might be helpful to include examples of what are considered "alternative" backends. Really, just listing the backends that are distributed with Airflow, but are ignored by this command.